src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberFieldSymbol.cs

```diff
@@ -323,7 +323,7 @@ public sealed override bool HasInitializer
             get { return _hasInitializer; }
         }
 
-        public VariableDeclaratorSyntax VariableDeclaratorNode
+        private VariableDeclaratorSyntax VariableDeclaratorNode
```

Comment:
> Generally, as a matter of software engineering hygiene, it is better to keep API surface area between components as small as possible.

I agree with that in general. However, SourceFixedFieldSymbol derives from this type and now it has an implicit contract with the base instead of an explicit one. I believe that an explicit contract is better than an implicit one. It makes dependency clear, code more readable and easier to understand. Consider changing accessibility of this method to protected and reverting the change in SourceFixedFieldSymbol.cs

---
In reply to: [80816186](https://github.com/dotnet/roslyn/pull/13999#discussion_r80816186) [](ancestors = 80816186,80815616)